python/langsmith/evaluation/evaluator.py

```diff
@@ -632,3 +636,70 @@ def comparison_evaluator(
 ) -> DynamicComparisonRunEvaluator:
     """Create a comaprison evaluator from a function."""
     return DynamicComparisonRunEvaluator(func)
+
+
+def _normalize_evaluator_func(
```

Comment:
might be nice to add like a couple unit tests on this to make it obvious it's working